### PR TITLE
replace create_text node options with attribute

### PR DESCRIPTION
**Pull request by msh-odoo**
PURPOSE
The option "create_text" implies that the text cannot be translated as it is passed as node options

SPEC
Pass create_text as field node attribute so that it is parsed by translate.py

TASK 1923433

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: odoo/addons/base/rng/common.rng [hunk 1]
```diff
@@ -267,6 +267,7 @@
             <rng:optional><rng:attribute name="write_field" /></rng:optional>
             <rng:optional><rng:attribute name="text" /></rng:optional>
             <rng:optional><rng:attribute name="optional" /></rng:optional>
+            <rng:optional><rng:attribute name="create_text"/></rng:optional>
```
Comment (odony) [inline odoo/addons/base/rng/common.rng:270]: `create_text` is unclear, this looks like a label for the "Add" button to me.
I know the option already existed, but it's time to change it if you make it part of the RNG ;-)
So maybe `add-label`? Also note that the majority of attributes are using hyphens as separators, let's aim for more consistency, not less.

Comment (msh-odoo) [reply odoo/addons/base/rng/common.rng:270]: @odony 👍

**Comment by SimonGenin** [comment]
@msh-odoo You'll need to rebase and check if create_text has not been used at new places since the task is quite old. 
@aab-odoo I kinda agree with odo. If create_text is the label of the button, we could rename it to something like create-label. For the hyphen remark I'm not sure as JS handles _ better than -. But attributes should use hyphens.

**Comment by aab-odoo** [comment]
@SimonGenin I think `add-label` is fine.

**Comment by SimonGenin** [comment]
@msh-odoo Let's go for add-label then.

**Review by SimonGenin — CHANGES_REQUESTED** [review]
@msh-odoo With a quick search in vscode, I see there is a least still two instances of create_text. 
One in community/addons/stock/views/stock_picking_views.xml and one in enterprise/marketing_automation/views/marketing_campaign_views.xml

**Comment by msh-odoo** [comment]
@SimonGenin I have done changes.

**Comment by xmo-odoo** [comment]
I'm not sure about `add-label`, at first glance it made me think this was modifying something like the field's label (`@string`) but it's instead the label for the [create]/[add] button of o2m/m2m fields yes?

**Comment by SimonGenin** [comment]
@xmo-odoo We feel that add-label is fine on our end

**Comment by SimonGenin** [comment]
@msh-odoo Can you update the commit message with the addons modified.

**Comment by msh-odoo** [comment]
@SimonGenin Done

**Comment by SimonGenin** [comment]
@robodoo r+

**Comment by SimonGenin** [comment]
@robodoo r+

**Pull request closed without merging**